src/main/java/core/basesyntax/service/impl/FileWriterImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.Writer;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class FileWriterImpl implements Writer {
+    @Override
+    public void writeToFile(String resultingReport, String path) {
+        try {
+            BufferedWriter writer = new BufferedWriter(new FileWriter(path));
```

Comment:
Consider using a try-with-resources statement to automatically close the `BufferedWriter`. This ensures that the resource is closed properly even if an exception occurs, improving resource management.